charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/rbac-csi-azuredisk-node.yaml

```diff
@@ -10,15 +10,27 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "list"]
     {{- if .Values.azuredisk.useV2Driver}}
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
```

Comment:
Since we're extending access to beyond that needed to access secrets, it would make sense to change the RBAC role name to `azuredisk-csi-node-role`

Reply:
Done :)